### DISCUSSION TITLE
Model not displaying when entering AR mode the first time

### DISCUSSION THREAD

**Discussion by SamAdobe** [google/model-viewer#4465] [Q&A]
Hi All,
A few times when I've entered AR mode from the main astronaut demo on https://modelviewer.dev/, the model never appears in my space. Exiting and re-entering AR mode fixes the issue. I didn't see anything in the console, but of course I can't reproduce it with a screen recorder attached.  I feel bad submitting a bug report without any concrete data to back it up, so has anyone else seen something similar?

Chrome 117.0.5938.60 on Android 13, Pixel 6

**Comment by elalish** [comment]
No, but ARCore is a mysterious beast and the fact it work as often as it does is a small miracle.
